### DISCUSSION TITLE
systemd support

### DISCUSSION THREAD

**Discussion by marcosps** [arighi/virtme-ng#193] [Ideas]
I'm sure that this was already being requested, but I couldn't find any issues opened/closed about it, or a discussion. And I'm also sure that this can be useful for more people, so I'm leaving a discussion open about it :)

For my usecase, I'm currently using virtme-ng to test reproducer for CVEs for different kernels. My workflow is that I have a more then 60 downloaded kernels, and I use vng to boot these kernels and run scripts to check if the kernels are vulnerable or not. It works nicely, even on different architectures. The issue is that sometimes I need to start services using systemctl, so a error message appears:

```
System has not been booted with systemd as init system (PID 1). Can't operate.
```

Just my 2 cents :)

**Comment by rbmarliere** [comment]
Basic support implemented in https://github.com/arighi/virtme-ng/pull/280. We might continue discussing here for improvements!

**Comment by marcosps** [comment]
Given that #280 was merged I think that we can close this discussion! Thanks again @rbmarliere !
